schematic/__init__.py

```diff
@@ -26,52 +36,116 @@
 logger = logging.getLogger(__name__)
 
 
-# borrowed from: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/tests/integration/conftest.py
-class FileSpanExporter(ConsoleSpanExporter):
-    """Create an exporter for OTEL data to a file."""
+def create_telemetry_session() -> requests.Session:
+    """
+    Create a requests session with authorization enabled if environment variables are set.
+    If no environment variables are set, the session will be created without authorization.
+
+    Returns:
+        requests.Session: A session object with authorization enabled if environment
+            variables are set. If no environment variables are set, the session will be
+            created without authorization. If no telemetry export format is set, None
+            will be returned.
+    """
+    tracing_export = os.environ.get("TRACING_EXPORT_FORMAT", None)
+    logging_export = os.environ.get("LOGGING_EXPORT_FORMAT", None)
+    if not (tracing_export or logging_export):
+        return None
+
+    session = requests.Session()
+    static_otlp_headers = os.environ.get("OTEL_EXPORTER_OTLP_HEADERS", None)
+    if static_otlp_headers:
+        logger.info(
+            "Using static OTLP headers set in environment variable `OTEL_EXPORTER_OTLP_HEADERS`."
+        )
+        return session
```

Comment:
This section is how developers running schematic locally will be able to export telemetry data with an auth header. The `OTEL_EXPORTER_OTLP_HEADERS` is built into the OTEL library. The library handles adding whatever is set in that environment variable to each outgoing request. Because of this i am just returning an initialized `requests.Session()` here without any additional configuration.

Reply:
See README.md for how developers would be using this.